app/assets/v2/js/grants/_new.js

```diff
@@ -277,7 +277,26 @@ Vue.mixin({
           _alert(err.message, 'danger');
         }
       });
-    }
+    },
+    onBlur(event) {
```

Comment:
We should rename this to reveal intent. Other elements may need to respond to blur and this could lead to confusion. `validateTwitterHandle` or something like that is a better name